plugins/tracker-resources/src/components/CreateIssue.svelte

```diff
@@ -113,51 +138,19 @@
         reports: 0,
         childInfo: []
       }
-    : toIssue(
-      {
-        title: '',
-        description: '',
-        assignee,
-        project,
-        sprint,
-        number: 0,
-        rank: '',
-        status: '' as Ref<IssueStatus>,
-        priority,
-        dueDate: null,
-        comments: 0,
-        subIssues: 0,
-        parents: [],
-        reportedTime: 0,
-        estimation: 0,
-        reports: 0,
-        childInfo: []
-      },
-      draft
-    )
+    : toIssue(defaultIssue, draft)
+
+  $: {
+    defaultIssue.workDayLength = currentTeam?.workDayLength ?? WorkDayLength.EIGHT_HOURS
+    defaultIssue.defaultTimeReportDay = currentTeam?.defaultTimeReportDay ?? TimeReportDayType.PreviousWorkDay
+    object.workDayLength = defaultIssue.workDayLength
+    object.defaultTimeReportDay = defaultIssue.defaultTimeReportDay
+  }
 
   function resetObject (): void {
     templateId = undefined
     template = undefined
-    object = {
-      title: '',
-      description: '',
-      assignee,
-      project,
-      sprint,
-      number: 0,
-      rank: '',
-      status: '' as Ref<IssueStatus>,
-      priority,
-      dueDate: null,
-      comments: 0,
-      subIssues: 0,
-      parents: [],
-      reportedTime: 0,
-      estimation: 0,
-      reports: 0,
-      childInfo: []
-    }
+    object = defaultIssue
```

Comment:
```suggestion
    object = {...defaultIssue}
```

Over wise create next one will use previous values.

Reply:
Done.